core/src/replay_stage.rs

```diff
@@ -2515,9 +2519,10 @@ impl ReplayStage {
         root_slot: u64,
         leader: &Pubkey,
         subscriptions: &Arc<RpcSubscriptions>,
+        vote_only_bank: bool,
     ) -> Bank {
         subscriptions.notify_slot(slot, parent.slot(), root_slot);
-        Bank::new_from_parent(parent, leader, slot)
+        Bank::new_from_parent_with_vote_only(parent, leader, slot, vote_only_bank)
```

Comment:
might be good to log a warning and emit a datapoint whenever this happens so that we can track it in the metrics

Reply:
added counter `replay_stage-new_vote_only_bank` for each vote-only bank creation, another counter `banking_stage-loop-stats.vote_only_bank_dropped_tx_count` tracks the number of non-vote transactions dropped due to vote-only